zhaquirks/tuya/ts001x.py

```diff
@@ -363,6 +364,85 @@ class Tuya_Double_No_N(TuyaSwitch):
     }
 
 
+class Tuya_Double_No_N__TZ3000_18ejxno0(TuyaSwitch):
```

Comment:
Don't reference the device in the class name.
Not sure which name can fit here. Maybe something like `Tuya_Double_No_N_Plus`?